src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriteScvService;
+
+public class ReportServiceImpl implements ReportService {
+    private FruitTransactionDao fruitTransactionDao;
+    private WriteScvService writeScvService;
+    private final String nameReportFile = "report.csv";
+    private StringBuilder data;
+
+    public ReportServiceImpl(FruitTransactionDao fruitTransactionDao,
+                             WriteScvService writeScvService) {
+        this.fruitTransactionDao = fruitTransactionDao;
+        this.writeScvService = writeScvService;
+    }
+
+    @Override
+    public void createReport() {
```

Comment:
Method should return the report
```suggestion
    public String createReport(List<FruitTransaction> fruitTransactions) {
```